games-misc/cowsay/cowsay-3.04.recipe

```diff
@@ -5,8 +5,8 @@ HOMEPAGE="https://github.com/tnalpgge/rank-amateur-cowsay"
 COPYRIGHT="1999-2000 Tony Monroe"
 LICENSE="GNU GPL v3"
 REVISION="1"
-SOURCE_URI="$HOMEPAGE/archive/$portName-$portVersion.tar.gz"
-CHECKSUM_SHA256="9a32cba0b561719e1302398857124fdf4dd21a7623c017eaadbf31277f9274af"
+SOURCE_URI="$HOMEPAGE/archive/cowsay-$portVersion.tar.gz"
```

Comment:
It's not strictly required in this case, since the architecture was `any`.

Reply:
Oh, I just changed it since there's no alternative to the value, and `$portName` isn't shorter than `cowsay`.